src/script.ts

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
the shorthand copyright notice would be sufficient here too:

```
/* Copyright (c) 2017-2019 Environmental Systems Research Institute, Inc.
 * Apache-2.0 */
```

Reply:
I did that for all the source files